mitodl/ocw-to-hugo
# refactor master -> parsed and uid -> short_url

#### Pre-Flight checklist


#### What are the relevant tickets?
Related to https://github.com/mitodl/ocw-data-parser/issues/76
Closes https://github.com/mitodl/ocw-to-hugo/issues/112

#### What's this PR do?
This PR refactors `ocw-to-hugo` to expect files to begin with the course `short_url` instead of `uid` and end with `_parsed.json` instead of `_master.json`.  This PR is 1 of 2 PR's needed to unblock https://github.com/mitodl/ocw-data-parser/pull/78.

#### How should this be manually tested?
Use the output from [this](https://github.com/mitodl/ocw-data-parser/pull/78) `ocw-data-parser` PR to generate some markdown and ensure that there are no changes in the results vs master.
